docs/troubleshooting/README.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting
+
+This page provides tips and tricks for troubleshooting
+
+## Tenant Controller or Run Controller are unresponsible
+
+When either Run Controller or Tenant Controller are unresponsibe a thread dump can be triggered by
+sending `SIGQUIT` to the corresponding processes:
+
+```bash
+pkill -QUIT -f '<PATTERN>'
+```
+
+where `PATTERN` depicts the application (`steward-runctl`,`steward-tenantctl`).
```

Comment:
Jjust a reminder: in case we decide to change the name [here](https://github.com/SAP/stewardci-core/pull/260) we need to adjust the application names also here.

Reply:
Other PR has been merged